cmd/nvidia-ctk/runtime/configure/configure.go

```diff
@@ -120,6 +129,12 @@ func (m command) build() *cli.Command {
 			Usage:       "the config mode for runtimes that support multiple configuration mechanisms",
 			Destination: &config.mode,
 		},
+		&cli.StringFlag{
+			Name:        "config-source",
+			Usage:       "the source to retrieve the container runtime configuration",
```

Comment:
Do we want to indicate the valid values for this flag in the usage string? E.g. [ file | command]